stdlib/test/builtin/test_float_literal.mojo

```diff
@@ -54,6 +54,25 @@ def test_floor():
     assert_equal(FloatLiteral.__floor__(neg_inf), neg_inf)
 
 
+def test_round():
+    assert_equal(FloatLiteral.__round__(1.5), 2.0)
```

Comment:
Can you please add a test for `2.5`. In `Python` that would round to 2, but IIRC our logic is currently always rounding away from zero.

Reply:
Great catch! Thanks! Fixed here, adding the proper tests: https://github.com/msaelices/mojo/commit/46149cafb9c23d64b7139c79874d5799cc479535

Reply:
I still don't see a test for `2.5`. Maybe you forgot to commit/push that patch?

Reply:
I thought you meant to have a `x.5` test, no matter if it's 1.5 or 2.5. Added here: https://github.com/msaelices/mojo/commit/757f24f153ed07e00c0620b87a79f7333d5f31f5